src/balena-utils.ts

```diff
@@ -146,6 +146,29 @@ export async function getReleaseByTags(
 	};
 }
 
+// https://www.balena.io/docs/reference/sdk/node-sdk/#balena.models.release.get
+export async function getReleaseVersion(releaseId: number): Promise<string> {
+	core.debug(`Getting raw version for release ID ${releaseId}`);
+
+	await balena.auth.loginWithToken(
+		core.getInput('balena_token', { required: true }),
+	);
```

Comment:
this makes me realize we should think of a better way to authenticate the SDK once...we can do it in another PR